yaml_test_runner/src/step/do.rs

```diff
@@ -122,7 +119,6 @@ impl Do {
             tokens.append(quote! {
                 let allowed_warnings = vec![#(#allowed),*];
                 let warnings: Vec<&str> = response.warning_headers()
-                    .filter(|w| !w.starts_with("[types removal]") && !allowed_warnings.iter().any(|a| w.contains(a)))
```

Comment:
Is removing the `allowed_warnings` filtering intended here? If so, then likely the `let allowed_warnings = ...` above can be removed

Reply:
That was my bad! Added the `allowed_warnings` filter back.